### PR TITLE
Theme tweaks & sticky footer

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: plonetheme/barceloneta/theme/index.html [hunk 1]
```diff
@@ -13,22 +13,21 @@
...
+  <div class="outer-wrapper flex-shrink-0">
+    <header id="content-header" class="container d-flex pt-3 flex-column justify-content-between align-items-md-start flex-md-row">
 
-      <div id="portal-top" class="container">
+      <div id="portal-top" class="d-flex">
       </div>
 
-      <div id="portal-logo-wrapper" class="container d-flex">
-        <div id="portal-logo"></div>
+      <div id="portal-logo-wrapper" class="pb-3 d-flex justify-content-between align-items-center order-md-first w-auto">
```
Comment (petschki) [inline plonetheme/barceloneta/theme/index.html:26]: why does the logo wrapper need `justify-content-between` and `align-items-center` ?

Comment (agitator) [reply plonetheme/barceloneta/theme/index.html:26]: some reponsive issue... or can one be removed?

Comment (petschki) [reply plonetheme/barceloneta/theme/index.html:26]: yes right ... wrapper classes are needed for mobile view. 👍🏼

**Review by petschki — APPROVED** [review]

**Comment by petschki** [comment]
This likely needs some testfixes 🧐

File: plonetheme/barceloneta/theme/rules.xml [hunk 2]
```diff
@@ -30,21 +30,30 @@
...
-  <replace css:theme="#portal-searchbox" css:content="#portal-searchbox" mode="raw" />
+  <!-- set aria-label for toggler button -->
   <copy attributes="aria-label" css:theme=".navbar-toggler" css:content=".navbar-toggler" />
-  <!-- <replace css:theme="button.navbar-toggler" css:content="button.navbar-toggler" mode="raw" /> -->
-
-
+  <!-- move searchbox into place -->
+  <replace css:theme-children="#portal-searchbox" css:content-children="#portal-searchbox" />
+  <!-- set bs styling for search button -->
+  <replace css:content=".searchButton">
```
[the remark this answers is not in the document]
Comment (petschki) [reply plonetheme/barceloneta/theme/rules.xml:42]: I'm also not a fan of doing this in XSLT

**Review by MrTango — APPROVED** [review]
LGTM

**Pull request merged**